### PR TITLE
[ADD] Two new options for chain length

### PR DESCRIPTION
**Pull request by SandrinePDR**
## References

_None_

## About The Pull Request

Added two new options for the chain length of the ankle cuffs
- short
- clipped together

## Changelog

Authored by: Claudia & Sandrine

```md
Assets:
- [Leather ankle cuffs] Two new options for chain length
```

## Checklist

- [X] The change has been tested locally
- [X] Licensing information and credits were filled in/modified for added/modified assets
- [X] I understand this change is licensed based on [Pandora's Asset Licensing](https://github.com/Project-Pandora-Game/pandora-assets/blob/master/ASSET_LICENSING.md) information

### CODE REVIEW

File: src/assets/foot_restraints/leather_ankle_cuffs/leather_ankle_cuffs.asset.ts [hunk 1]
```diff
@@ -110,6 +110,48 @@ DefineAsset({
...
+				{
+					id: 'clipped',
+					name: 'Clipped together',
+					properties: {
+						poseLimits: {
+							options: [
+								{
+									legs: 'standing',
+									bones: {
+										leg_r: 2,
```
Comment (ClaudiaMia) [inline src/assets/foot_restraints/leather_ankle_cuffs/leather_ankle_cuffs.asset.ts:137]: 2/2 does not only work for standing, but also for sitting in my tests.

I think there is no reason to not forbid kneeling in the clipped together state.

**Review by ClaudiaMia — CHANGES_REQUESTED** [review]
Great idea! Found one small thing.

File: src/assets/foot_restraints/leather_ankle_cuffs/leather_ankle_cuffs.asset.ts (hunk 1 above)
Comment (SandrinePDR) [reply src/assets/foot_restraints/leather_ankle_cuffs/leather_ankle_cuffs.asset.ts:137]: The clipped state in kneeling requires bones of 6/6 (lines 142ff in the same file)

File: src/assets/foot_restraints/leather_ankle_cuffs/leather_ankle_cuffs.asset.ts [hunk 2]
```diff
@@ -110,6 +110,48 @@ DefineAsset({
...
+										leg_l: 2,
+									},
+								},
+								{
+									legs: 'kneeling',
+									bones: {
+										leg_r: 6,
+										leg_l: 6,
+									},
```
Comment (SandrinePDR) [inline src/assets/foot_restraints/leather_ankle_cuffs/leather_ankle_cuffs.asset.ts:146]: The kneeling position for clipped cuffs requires bones of 6/6

**Review by ClaudiaMia — APPROVED** [review]
LGTM!

**Pull request merged**